lib/srv/exec.go

```diff
@@ -468,7 +478,7 @@ func getDefaultEnvPath(uid string, loginDefsPath string) string {
 	// ENV_PATH first, then the default path.
 	if uid == "0" {
 		if envSuPath == defaultEnvPath {
-			return envPath
+			return defaultEnvPath
 		}
 		return envSuPath
```

Comment:
is this equivalent to just `return envSuPath`?

Reply:
Yes, I think so.